src/main/java/no/nav/veilarbaktivitet/client/ArenaAktivitetClientImpl.java

```diff
@@ -93,6 +88,17 @@ static Date parseDato(String konfigurertDato) {
         }
     }
 
+    @Override
+    public HealthCheckResult checkHealth() {
```

Comment:
Stemmer det at det her er den eneste endringen som ikke er refaktorering av det vi allerede har? Ser ikke helt hvor "bruk systembruker for kall mot arena" endringen kommer inn

Reply:
`configureStsForSystemUser` gjør at SOAP kallet blir gjort med systembruker.  Men jeg tror vi kanskje legger denne PRen på is siden det ser ut som det ikke er støtte for systembruker i det hele tatt siden kall fra veilarboppfolging feiler